projects/plugins/jetpack/extensions/blocks/conversation/example.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * WordPress dependencies
+ */
+
+ import { __ } from '@wordpress/i18n';
+
+const participants = [
+	{
+		participantSlug: 'participant-0',
+		participant: 'Rosalind',
```

Comment:
```suggestion
		participant: __( 'Rosalind', 'jetpack' ),
```

Reply:
I think we shouldn't translate the names.